src/lib/viewers/controls/annotations/AnnotationsTargetedTooltip.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import TargetedClickThroughGuideTooltip from 'box-ui-elements/es/features/targeting/TargetedClickThroughGuideTooltip';
```

Comment:
It looks like this relies on `react-tether`. Can we ensure that we're using the same version BUIE expects? Can we also do a comparison of the production bundle size before and after this change?

Reply:
I updated react-tether to 1.0.5. I will compare the bundle sizes now...

Reply:
It looks like Preview.js went up about 100 Kib and Preview.css 10 Kib. Do you think that sounds reasonable?
<img width="452" alt="Screen Shot 2021-05-03 at 2 17 54 PM" src="https://user-images.githubusercontent.com/75504287/116934971-870f8300-ac1a-11eb-9b7a-4832c493fb2d.png">

Reply:
Hm, that seems like a big jump given the nature of the change. Can we use the bundle analyzer to make sure we're not pulling in code we shouldn't be, like another version of React or something? 

We should also check the built CSS file to make sure we're not bringing in extra global styles from the Button components in GuideTooltip. I think we've run into issues around that before.

Reply:
<img width="235" alt="Screen Shot 2021-05-04 at 8 59 06 AM" src="https://user-images.githubusercontent.com/75504287/117035800-9bef2380-acb9-11eb-86c7-a439a10aeeab.png">
preview.js is now at 755 KiB. I ran the analyzer and see that the Tooltip Component in BUIE is about 15.8 KiB, therefore I think the Tooltip component + React Tether make up most of the 54 additional KiB. I'm not sure what's included in the 10 KiB extra for preview.css. Do you know how to see what styles are included in the webpack analyzer plugin?